packages/docs/source/components/status.md

```diff
@@ -0,0 +1,211 @@
+# Status
+
+Status is used to keep users informed, by providing appropriate feedback on particular concepts or entities. For Okta, status can be used to display overall operational status all the way down to very granular concepts, like user status.
+
+## Usage
+
+Status should be used to communicate the state of a discrete item, such as a server or individual process. Both labeled and unlabeled variants are acceptable, but should follow the associated guidelines below.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-success">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value">
+        System operational
+      </dd>
+    </dl>
+  </div>
+
+  ```html
+  <dl class="ods-status is-ods-status-success">
+    <dt class="ods-status--label">
+      Server status
+    </dt>
+    <dd class="ods-status--value">
+      System operational
+    </dd>
+  </dl>
+  ```
+</figure>
+
+Labels may be hidden by applying the `.is-ods-status-label-hidden` class. If a label is not visually present, ensure that appropriate context is communicated by the Status itself and close proximity to the content that it’s supporting.
+
+Even if the label is hidden, it must be populated to ensure appropriate context for users of assistive technology.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-success is-ods-status-label-hidden">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value">
+        System operational
+      </dd>
+    </dl>
+  </div>
+
+  ```html
+  <dl class="ods-status is-ods-status-success is-ods-status-label-hidden">
+    <dt class="ods-status--label">
+      Server status
+    </dt>
+    <dd class="ods-status--value">
+      System operational
+    </dd>
+  </dl>
+  ```
+</figure>
+
+## Variants
+
+There are four variants of Status available: Neutral, Success, Caution, and Danger.
+
+### Neutral
+
+Neutral Statuses are gray and should be used to indicate states like Paused, Not started, or Queued.
```

Comment:
This is interesting, neutral appears to be the "default" style, should we communicate that?